deploy/buildtools/buildafi.py

```diff
@@ -211,7 +281,7 @@ def aws_create_afi(global_build_config, buildconfig):
         rootLogger.info("Resulting AFI: " + str(afi))
 
     rootLogger.info("Waiting for create-fpga-image completion.")
-    results_build_dir = """{}/results-build/{}/""".format(ddir, builddir)
+    results_build_dir = """{}/""".format(local_results_dir)
```

Comment:
On another note, is there some preference to use `"""` vs `"` for strings?

Reply:
I guess my personal opinion is to basically do the same as Scala: triples for multiline literals and when escaping quotes becomes cumbersome, otherwise just one `"`.

Reply:
I agree with what David said, only use triples for multiline.  I would add that I prefer using format strings instead of calls to format since we've moved into Python3 (i.e. `f"{local_results_dir}/"` ).  yeah, I know we're going to remove it in this case but wanted to give a concrete format string example.

Reply:
Since making this comment, I've seen other examples where double and single quotes were both used literally inside of triple double-quotes and that's another reasonable use for the triple quotes.